addons/mail/static/src/owl/components/chatter/chatter.js

```diff
@@ -14,52 +15,66 @@ class Chatter extends Component {
      */
     constructor(...args) {
         super(...args);
-        this.state = useState({ isAttachmentBoxVisible: false });
+        this.state = useState({
+            isComposerLog: false,
+            isAttachmentBoxVisible: false,
+            isComposerVisible: false,
+        });
         this.storeDispatch = useDispatch();
         this.storeGetters = useGetters();
-        this.storeProps = useStore((state, props) => {
-            const thread = this.storeGetters.thread({
-                _model: props.model,
-                id: props.id,
-            });
+        this.storeProps = useStore((state, { chatterLocalId }) => {
```

Comment:
And then in template when you need threadLocalId in template you use `storeProps.chatter.thread.localId` ? And you can consider thread is always defined or you add a `t-if="storeProps.chatter.thread` ?

Reply:
I don't think you need to use thread object in template. Maybe `threadLocalId`, but then it's just `storeProps.chatter.threadLocalId`

Reply:
I also need to know if the thread is temporary or not, since it's computed based on the fact that `res_id` is set or not (on chatter object in store) is it better to use :
- `!storeProps.chatter.res_id`
- or put `thread` in the storeProps and use `storeProps.thread.isTemporary
- or "compute it" and directly put `threadIsTemporary` in storeProps ?

Reply:
I prefer 2nd option :)